whitesource/client.py

```diff
@@ -64,41 +64,47 @@ def save_project_tag(self, projectToken: str, key: str, value: str):
             json=body,
         )
 
-    def post_project(
+    async def ws_project(
```

Comment:
I would prefer a more expressive name. such as `upload_to_project` or sth. similar (the previous name was a bit better, but also not great, as it was named after the technical detail of the underlying http-post request. a good name should not have to be changed just because the communication method changes)

Reply:
Sure, **upload_to_project** seems fine